yarn-project/circuits.js/src/abis/__snapshots__/abis.test.ts.snap

```diff
@@ -51,10 +51,10 @@ Fr {
 exports[`abis wasm bindings computes a function selector 1`] = `
 {
   "data": [
-    169,
-    5,
-    156,
-    187,
```

Comment:
@charlielye I got close, all tests pass except this diff. This doesn't seem right - I couldn't trace it before logging off

Reply:
@suyash67 any idea?